app/pages/admin/project-status.jsx

```diff
@@ -106,7 +111,10 @@ class ProjectStatus extends Component {
     const defaultWorkflowId = this.state.project.configuration.default_workflow;
 
     if (defaultWorkflowId === workflow.id && workflow.active) {
-      this.setState({ dialogIsOpen: true });
+      this.setState({
+        defaultWorkflowId,
```

Comment:
This is redundant with the `default_workflow` already a property on the project resource which is already stored in the component state.